sdk/core/azure-core-tracing-opentelemetry/tests/test_servicebus_live.py

```diff
@@ -0,0 +1,202 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import pytest
+
+from azure.servicebus import ServiceBusClient, ServiceBusMessage
+from opentelemetry.trace import SpanKind
+from opentelemetry.sdk.trace import ReadableSpan
+
+
+class TestServiceBusTracing:
```

Comment:
We had an issue in SB about some of the extra params that we pass to logs being None/empty string, and that going against the otel spec. Not sure if you could/want to test that here? https://github.com/Azure/azure-sdk-for-python/issues/32190#issuecomment-1743752919

Reply:
The `extra` param seems to be more pertinent to OTel logging (as opposed to Tracing). But yea, good call on the null/empty string checks for attributes in general. I can add a check to make sure that all attributes of a span or not empty string or None.